### PR TITLE
Some more spooky? names!

### PR DESCRIPTION
**Pull request by TSampson92**

### CODE REVIEW

File: CSSBot/Services/TheSpookening/spooky.json [hunk 1]
```diff
@@ -78,7 +78,13 @@
     "{1}",
     "💡 moth 💡",
     "slaps roof of {0}",
-    "cssbot's codebase"
+    "cssbot's codebase",
+	"PR Author that auto resolves comments",
```
Comment (Chris-Johnston) [inline CSSBot/Services/TheSpookening/spooky.json]: This one might be too long?

**Review by Chris-Johnston — CHANGES_REQUESTED** [review]
MORE

**Comment by TSampson92** [comment]
> 
> 
> MORE



> 
> 
> MORE

**Comment by Chris-Johnston** [comment]
sure

**Review by Chris-Johnston — APPROVED** [review]

**Pull request merged**